pkg/restic/common.go

```diff
@@ -79,6 +70,8 @@ func GetPodSnapshotAnnotations(obj metav1.Object) map[string]string {
 
 // SetPodSnapshotAnnotation adds an annotation to a pod to indicate that
 // the specified volume has a restic snapshot with the provided id.
+// Deprecated: we will stop using pod annotation to determine restic
```

Comment:
This comment isn't accurate - should say that we're no longer using pod annotations to record restic snapshot IDs after they're taken.